build_image

```diff
@@ -196,7 +196,11 @@ split_ver "${FLATCAR_VERSION_ID}" SPLIT
 tee "${BUILD_DIR}/version.txt" <<EOF
 FLATCAR_BUILD=${SPLIT[0]}
 FLATCAR_BRANCH=${SPLIT[1]}
-FLATCAR_PATCH=${SPLIT[2]}
+EOF
+if [[ ${#SPLIT[@]} -eq 3 ]]; then
```

Comment:
probably could end up in common.sh

Reply:
The whole file creation part here? The version.txt is only created once for an image.

Reply:
sorry, was thinking about the version comparisons.